### PR TITLE
Correct medium security hotspots

### PR DESCRIPTION
**Pull request by cfkoehler**
Address two medium security hotspots. Not really vulnerabilities in the application with the way they are used but it get's flagged on scans. If the changes are not preferred then I can look into a way of suppressing the scans.

DropOffUtil random change: https://rules.sonarsource.com/java/RSPEC-2245
FilePickUp hash change: https://rules.sonarsource.com/java/RSPEC-4790

### CODE REVIEW

File: src/main/java/emissary/pickup/file/FilePickUpClient.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public FilePickUpClient(InputStream configInfo) throws IOException {
      */
     protected void configurePlace() {
         try {
-            digest = MessageDigest.getInstance("MD5");
+            digest = MessageDigest.getInstance("SHA-256");
```
Comment (dev-mlb) [inline src/main/java/emissary/pickup/file/FilePickUpClient.java:92]: Should we make this a configurable value?

Comment (cfkoehler) [reply src/main/java/emissary/pickup/file/FilePickUpClient.java:92]: I don't know where we would need to have it different. But it is easy enough to add so will do that.

Comment (dev-mlb) [reply src/main/java/emissary/pickup/file/FilePickUpClient.java:92]: If it is not needed, don't worry about it. I didn't know if you'd need to change it internally without having to do a code change.

**Comment by cfkoehler** [comment]
- [ ] Made the most recent commit in Gitub's web interface. Need to pull down and test.

**Review by dev-mlb — APPROVED** [review]

**Review by jpdahlke — APPROVED** [review]

**Pull request merged**